pom.xml

```diff
@@ -841,6 +844,391 @@
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <!-- This profile is activated when Java 11 or later is used to build -->
+    <profile>
+      <id>java11-mr-build</id>
+      <activation>
+        <jdk>[11,</jdk>
+        <file>
+          <exists>${basedir}/src/main/java11</exists>
+        </file>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>compile-java11</id>
+                <phase>compile</phase>
+                <goals>
+                  <goal>compile</goal>
+                </goals>
+                <configuration>
+                  <release>11</release>
+                  <buildDirectory>${project.build.directory}</buildDirectory>
+                  <compileSourceRoots>${project.basedir}/src/main/java11</compileSourceRoots>
+                  <outputDirectory>${project.build.directory}/classes/META-INF/versions/11</outputDirectory>
+                  <additionalClasspathElements>
+                    <additionalClasspathElement>${project.build.directory}/classes/META-INF/versions/10</additionalClasspathElement>
+                    <additionalClasspathElement>${project.build.directory}/classes/META-INF/versions/9</additionalClasspathElement>
+                    <additionalClasspathElement>${project.build.outputDirectory}</additionalClasspathElement>
+                  </additionalClasspathElements>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <artifactId>maven-jar-plugin</artifactId>
+            <configuration>
+              <archive>
+                <manifestEntries>
+                  <Multi-Release>true</Multi-Release>
+                </manifestEntries>
+              </archive>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <!-- This profile is activated when Java 12 or later is used to test a project that supports Java 11 -->
+    <profile>
+      <id>java11-test</id>
+      <activation>
+        <jdk>[12,</jdk>
+        <property>
+          <name>java11.home</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>java11-test</id>
+                <phase>test</phase>
+                <goals>
+                  <goal>test</goal>
+                </goals>
+                <configuration>
+                  <jvm>${java11.home}/bin/java</jvm>
+                  <classesDirectory>${project.build.directory}/classes/META-INF/versions/10</classesDirectory>
```

Comment:
Should his be `12` with `11` added below in the `additionalClasspathElement`?

Reply:
This is fixed now.